docker-compose.yml

```diff
@@ -38,6 +38,23 @@ services:
     networks:
       - mynet
 
+  mongo-express:
```

Comment:
Nice work! But... This block depends on MongoDB, so it makes more sense for me to stay above it.

Reply:
@fernandohenriques fixed! I moved the service for Mongo Express as mentioned, take a look to see if it looks good, please.